src/transformState.js

```diff
@@ -4,8 +4,26 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const { type } = action;
```

Comment:
```suggestion
    const { type, extraData, keysToRemove } = action;
```
why didn't you destructure another keys